mne_bids/read.py

```diff
@@ -185,6 +186,28 @@ def _handle_participants_reading(participants_fname, raw,
     return raw
 
 
+def _handle_scans_reading(scans_fname, raw, bids_path, verbose=False):
+    """Read associated scans.tsv and set meas_date."""
+    scans_tsv = _from_tsv(scans_fname)
+
+    # get the row
+    data_fname = op.join(bids_path.datatype, bids_path.fpath.name)
```

Comment:
Will this work as intended on Windows?

Reply:
Not sure... what is the suggested way to do this? for 

`eeg/sub<blah>_eeg.edf`?

Reply:
Noticed CI has a windows, so I'll wait to see if it get's handled.

Reply:
why wouldn't `bids_path.datatype / bids_path.fpath.name` work? I thought that's one of the main syntactic sugars of pathlib.Path?

Reply:
Okay, I see - thanks!